client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -8,11 +8,14 @@
 </license>
 
 <template>
-  <div>
+  <div
+    :class="{ 'top-0 left-0 w-full h-full fixed z-[2200] bg-white': isFullscreen }">
```

Comment:
We have a predefined z-index for fixed elements. Also, we do not need to break attrs onto a new line if there is only one attr.

```suggestion
  <div :class="{ 'top-0 left-0 w-full h-full fixed z-fixed bg-white': isFullscreen }">
```

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/2979/commits/a842dfe8552b4c374486dcc41c97d3f971f4e2b2 and here https://github.com/demos-europe/demosplan-core/pull/2979/commits/603e652ef6732baa8afc34247685735dc15c8d69